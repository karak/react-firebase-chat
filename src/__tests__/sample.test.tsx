import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

describe('<a> component', () => {
  const element = <a href="http://www.facebook.com">Facebook</a>;

  describe('when shallow rendered', () => {
    const wrapper = shallow(element);

    it('has href', () => {
      expect(wrapper.find('a')).toHaveProp('href', 'http://www.facebook.com');
    });

    it('has text content', () => {
      expect(wrapper.find('a')).toHaveText('Facebook');
    });
  });

  it('retains snapshot', () => {
    const tree = renderer.create(element).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
