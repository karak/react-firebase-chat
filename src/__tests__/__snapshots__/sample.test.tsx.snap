// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<a> component retains snapshot 1`] = `
<a
  href="http://www.facebook.com"
>
  Facebook
</a>
`;
